ruby/cucumber-gherkin.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |s|
     'source_code_uri' => 'https://github.com/cucumber/gherkin/blob/main/ruby'
   }
 
-  s.add_runtime_dependency 'cucumber-messages', '>= 19.1.4', '< 22.1'
+  s.add_runtime_dependency 'cucumber-messages', '~> 22.0'
```

Comment:
Unless something materially changed about the messages, is there a reason not to increase the upper bound instead?

Reply:
As in permit 19.1 but increase the upper limit to say 23?

Currently the maximum version is 22.0 so any revision in the 22.x should be fine here. Just trying to reduce the support surface area. In theory we could keep this support area the same?

Reply:
Yes. Allowing a wider range allows us to make major releases of messages without also making major releases of Gherkin. 

This reduces the churn somewhat.

Reply:
I mean in theory nothing stops us from setting the upper limit to 30 by the same logic. Or having no upper limit.

Bundler flags things without an upper limit hence the twiddle key. In theory something could come in v23 which breaks things hence setting this value?

Reply:
Yeah I would think `'>= 19.1.4', '< 23'` should be good.

Reply:
I'll make that change now.